packages/emitter-framework/src/python/components/class-variable.tsx

```diff
@@ -0,0 +1,16 @@
+import { code } from "@alloy-js/core";
+import { usePythonNamePolicy } from "../name-policy.js";
+
+export interface ClassVariableProps {
+    name: string;
+    // TODO: Replace this with a component so we can handle imports?
+    type: string;
+}
+
+export function ClassVariable(props: ClassVariableProps) {
```

Comment:
I'm assuming these can be referenced, but symbol work here is todo?

Reply:
Correct.